src/utils/getTeams.test.ts

```diff
@@ -4,16 +4,16 @@ import { getTeams } from './getTeams';
 
 // Check tests/product-owners.yml for data used for tests here
 describe('getTeams', function () {
-  it('should return empty array if no teams were found', () => {
-    expect(getTeams('repo-does-not-exist', 'getsentry')).toEqual([]);
+  it('should return team-ospo if no teams were found', () => {
+    expect(getTeams('repo-does-not-exist', 'getsentry')).toEqual(['team-ospo']);
   });
 
   it('should return array with one team if repo does not have routing', () => {
-    expect(getTeams('test-ttt-simple', 'getsentry')).toEqual(['team-ospo']);
+    expect(getTeams('test-ttt-simple', 'getsentry')).toEqual(['team-issues']);
```

Comment:
Is the team selected at random?

Reply:
If you look at `test/product-owners.yml` you'll see
```
test-ttt-simple: team-issues
```

I could comment that if it makes it more clear

Reply:
I get that, but I'm wondering how that maps to a real repo with no routing - how does it pick the team in that case?

Reply:
Ah, that's done in security-as-code:
https://github.com/getsentry/security-as-code/blob/main/rbac/lib/product-owners.yml#L305

Then, it's synced to this repo when changes are made there